src/__tests__/elements-inside-label.js

```diff
@@ -0,0 +1,37 @@
+// query utilities:
+import {getByLabelText} from '../'
+import document from './helpers/document'
+
+function getExampleDOM() {
+  const div = document.createElement('div')
+  div.innerHTML = `
+    <label>Cars
+        <select>
+            <option>Volvo</option>
+            <option>Mercedes</option>
+        </select>
+    </label>
+    <label><span>States</span>
+        <select>
+            <option>Damaged</option>
+            <option>Good</option>
+        </select>
+    </label>
+    <label><input type="checkbox" value="1"> I agree to terms </label>
+  `
+  return div
+}
+
+test('elements inside label', () => {
+  const container = getExampleDOM()
+
+  const element1 = getByLabelText(container, 'Cars')
+  expect(element1.tagName).toBe('SELECT')
+
+  // this fails because getNodeText function returns incorrect result on JSDOM but it good for browser
+  // const element2 = getByLabelText(container, 'States');
+  // expect(element2.tagName).toBe('SELECT');
```

Comment:
This lines added temporarily added until switching jest to karma.

Reply:
> This lines added temporarily added until switching jest to karma.

Wait.... Are you attempting to migrate our tests to karma? If so then please don't take the time, it will not be merged.

If you want to add a few tests in karma and leave the existing tests as they are that's fine. The bill of the tests should be in jest, but I'd be happy to have a few tests that run in a real browser with karma.

Just want to make sure we're clear on expectations here.

Reply:
Ok I understand.